src/github/client/github-client-interceptors.ts

```diff
@@ -94,9 +94,9 @@ export const instrumentFailedRequest = (metricName: string, host: string) =>
 
 export const handleFailedRequest = (logger: Logger) =>
 	(error: AxiosError) => {
-		const { response, config, request } = error;
+		const { response, config  } = error;
 		const requestId = response?.headers?.["x-github-request-id"];
-		logger = logger.child({ res: response, config, req: request, err: error, requestId });
+		logger = logger.child({ requestId });
```

Comment:
Would it make sense to log request's URL + method and error message?

Reply:
the log.error is **returning, status, path, method**.

The err obj doesn't have a additional msg in this case :shakes-fist:
example error:
`Error executing Axios Request (POST https://api.github.com/app/installations/26528447/access_tokens): Request failed with status code 404
`

Ahh, but theres a couple after that missing it, will add as childs